docker-compose/cassandra-3.11/docker-compose-dev-mode.yml

```diff
@@ -33,18 +33,5 @@ services:
       - STARGATE_BRIDGE_HOST=coordinator
       - STARGATE_BRIDGE_PORT=8091
       - STARGATE_REST_PORT=8082
-  graphqlapi:
-    image: stargateio/graphqlapi:${SGTAG}
-    depends_on:
-      - coordinator
-    networks:
-      - stargate
-    ports:
-      - 9080:9080
-    mem_limit: 2G
-    environment:
-      - STARGATE_BRIDGE_HOST=coordinator
-      - STARGATE_BRIDGE_PORT=8091
-      - STARGATE_GRAPHQL_PORT=9080
```

Comment:
why removing the graphql from docker compose? we should still have it imo.. I know there is a ticket to add docs api now..

This applies to all docker related files below..